### PR TITLE
docs: Add rest-proxy docs

### PR DESCRIPTION
**Pull request by pvaneck**
Added some docs about rest-proxy that was added internally but not externally.
Also updated the deployment components resource usage chart and added some missing OVMS references.

### CODE REVIEW

**Comment by pvaneck** [comment]
Most original authors (from what I could tell) are listed as co-authors in the commit.

File: docs/configuration/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 System-wide configuration parameters can be set by creating a ConfigMap with name `model-serving-config`. It should contain a single key named `config.yaml`, whose value is a yaml doc containing the configuration. All parameters have defaults and are optional. If the ConfigMap does not exist, all parameters will take their defaults.
 
-The configuration can be updated at runtime and will take effect immediately. Be aware however that certain changes could cause temporary disruption to the service - in particular changing the service name, port, TLS configuration and/or headlessness.
+he configuration can be updated at runtime and will take effect immediately. Be aware however that certain changes could cause temporary disruption to the service - in particular changing the service name, port, TLS configuration, and/or headlessness, or enabling/disabling the REST inferencing port.
```
Comment (chinhuang007) [inline docs/configuration/README.md:5]: Seems an accidental typo "he"

Comment (pvaneck) [reply docs/configuration/README.md]: Good catch! fixed!

**Review by chinhuang007 — APPROVED** [review]
/lgtm

**Review by chinhuang007 — APPROVED** [review]
/lgtm

**Pull request merged**